src/shapes/rect.class.ts

```diff
@@ -77,7 +77,7 @@ const Rect = fabric.util.createClass(fabric.Object, /** @lends Rect.prototype */
     const x = -w / 2;
     const y = -h / 2;
     rx = rx ? Math.min(rx, w / 2) : 0;
-    ry = this.ry ? Math.min(ry, h / 2) : 0;
+    ry = ry ? Math.min(ry, h / 2) : 0;
```

Comment:
I meant 
```suggestion
    const ry = this.ry ? Math.min(this.ry, h / 2) : 0;
```

and remove `let ry`

Reply:
ok now i done it differently. is not really an issue.